docs/extending.md

```diff
@@ -3,5 +3,21 @@ id: extending
 title: Extending
 sidebar_label: Extending
 ---
+### Custom Phrases
+It is possible to create your own phrases which trigger cypress commands. In order to create a custom phrase, add something like the following to your `/support/step_definitions/your-component/feature.js` file:
```

Comment:
let's expand this section by offering a few examples of real phrases you can make and how it would handle variables. Also we should introduce the helpers here